cmd/stack_lint_dockerfile_stack.go

```diff
@@ -79,8 +79,12 @@ func lintDockerFileStack(log *LoggingConfig, stackPath string) (int, int) {
 			}
 		}
 		if !variableFound {
-			log.Warning.log("Missing ", variable)
-			stackLintWarningCount++
+			if optionalEnvironmentVariables[i] == "APPSODY_DEBUG_PORT" {
+				log.Warning.log("Missing APPSODY_DEBUG_PORT. If your stack supports the appsody debug command, please define this variable.")
```

Comment:
You could check for `APPSODY_DEBUG` to be set? And only through the error if so? you could also concatinate the errmg and pass it to `log.Warning.log("Missing ", variable, errMsgDebug)`

Reply:
This sounds like a better way of doing it, done 👍